.pipelines/nuget-publishing.yml

```diff
@@ -54,12 +45,8 @@ trigger: none
 stages:
 - template: stages/nuget-packaging-stage.yml
   parameters:
-    enable_linux_cpu: ${{ parameters.enable_linux_cpu }}
```

Comment:
No plan to support nuget linux release for upcoming release.

Reply:
Why? You mean the nuget package only contain Windows binaries?

Reply:
@yufenglee , can you please confirm? I am not in the project's scrums. So I do not have a full knowledge of it.

Reply:
So far, we have only validated and tested the C# API on Windows x64 machines. Since our initial users are going to want to use the C# API on windows machines, we prioritized that over making it available on linux as well.

Going forward, (probably for the next release), we will make C# API available for linux as well.
But for now, we will only need the nuget to contain binaries for Windows alone.